jujugui/static/gui/src/app/app.js

```diff
@@ -989,6 +989,15 @@ YUI.add('juju-gui', function(Y) {
       var state = this.state;
       var utils = views.utils;
       var charmstore = this.get('charmstore');
+      // Configure syntax highlighting for the markdown renderer.
+      marked.setOptions({
```

Comment:
So we don't have to trigger it to render after content has been added to the dom?